programs/drift/src/math/spot_withdraw.rs

```diff
@@ -155,15 +175,11 @@ pub fn calculate_availability_borrow_liquidity(spot_market: &SpotMarket) -> Drif
         spot_market.withdraw_guard_threshold.cast()?,
     )?;
 
-    let min_deposit_token = calculate_min_deposit_token(
-        spot_market.deposit_token_twap.cast()?,
-        spot_market.withdraw_guard_threshold.cast()?,
-    )?;
-
-    Ok(max_borrow_token
+    let borrow_limit = max_borrow_token
         .saturating_sub(borrow_token_amount)
-        .min(deposit_token_amount.saturating_sub(min_deposit_token))
-        .min(deposit_token_amount.saturating_sub(borrow_token_amount)))
+        .min(deposit_token_amount.saturating_sub(borrow_token_amount));
```

Comment:
is this just a redundancy guard, should never allow borrow > deposit through rest of system, right?